test/unit_tests/utils/LineChartComparison_test.js

```diff
@@ -0,0 +1,60 @@
+/* global before describe it */
+
+'use strict';
+
+require( 'jsdom-global' )();
+
+const chai = require( 'chai' );
+const mock = require( 'mock-require' );
+const expect = chai.expect;
+
+mock( 'highcharts/js/highstock', {
+  setOptions: () => { },
```

Comment:
linting: "Unexpected empty arrow function". (I'm not sure how necessary this is to fix.)